### ISSUE TITLE
[BUG] ecli-rs cannot be built successfully

### ISSUE THREAD

**Issue by try-agaaain** [eunomia-bpf/eunomia-bpf#138]
After using `wasm-bpf` to replace `wasm-runtime` in the project two days ago, the rust code under the ecli/ecli-rs folder has not been modified, resulting in the failure to successfully build ecli-rs. Therefore, the rust code and corresponding build files need to be corrected.

Expected result: After executing the `cd ecli/ecli-rs/ && make` command, a normal `ecli` executable file can be built.

**Comment by yunwei37** [comment]
Please provide some detail error message when reporting bugs. Are you facing the issue like this?

```sh
error: failed to run custom build command for `openssl-sys v0.9.80`

Caused by:
  process didn't exit successfully: `/home/yunwei/eunomia-bpf/ecli/ecli-rs/target/release/build/openssl-sys-ace4a9019edd3c26/build-script-main` (exit status: 101)
  --- stdout
  cargo:rustc-cfg=const_fn
  cargo:rustc-cfg=openssl
  cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_OPENSSL_LIB_DIR
```

**Comment by try-agaaain** [comment]
It seems that I have not encountered this error.

After building wasm-bpf and bpf-loader normally, the following error occurred:

```console
cargo build --release
    Compiling ecli v0.1.0 (/home/todo/intership/rs-ecli/ecli/ecli-rs)
error[E0428]: the name `iterator` is defined multiple times
      --> src/wasm_bpf_runner/wasm_bpf.rs:14551:1
       |
14549 | pub type iterator = std__Bit_iterator;
       | ----------------------------------------- previous definition of the type `iterator` here
14550 | pub type size_type = std_size_t;
14551 | pub type iterator = std__Bit_iterator;
       | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `iterator` redefined here
       |
       = note: `iterator` must be defined only once in the type namespace of this module

error[E0428]: the name `__hashtable` is defined multiple times
      --> src/wasm_bpf_runner/wasm_bpf.rs:14591:1
       |
14590 | pub type __hashtable = std__Hashtable<_RehashPolicy>;
       | -------------------------------------------------- ---- previous definition of the type `__hashtable` here
14591 | pub type __hashtable = std__Hashtable<_RehashPolicy>;
       | ^^^^ `__hashtable` redefined here
       |
       = note: `__hashtable` must be defined only once in the type namespace of this module

error[E0432]: unresolved imports `super::wasm_bpf::wasm_bpf_start`, `super::wasm_bpf::new_wasm_bpf`
   --> src/wasm_bpf_runner/wasm.rs:16:23
    |
16 | use super::wasm_bpf::{wasm_bpf_start, new_wasm_bpf};
    | ^^^^^^^^^^^^^^ ^^^^^^^^^^^^ no `new_wasm_bpf` in `wasm_bpf_runner::wasm_bpf`
    | |
    | no `wasm_bpf_start` in `wasm_bpf_runner::wasm_bpf`

error[E0412]: cannot find type `_Tp` in this scope
     --> src/wasm_bpf_runner/wasm_bpf.rs:1510:27
      |
1510 | pub static std_value: _Tp;
      | ^^^ not found in this scope

error[E0412]: cannot find type `_RehashPolicy` in this scope
     --> src/wasm_bpf_runner/wasm_bpf.rs:7268:48
      |
7268 | pub type std___uset_hashtable = std__Hashtable<_RehashPolicy>;
      | - ^^^^^^^^^^^^^^ not found in this scope
      | |
      | help: you might be missing a type parameter: `<_RehashPolicy>`
......

```
I know the third error:`error[E0432]: unresolved imports super::wasm_bpf::wasm_bpf_start, super::wasm_bpf::new_wasm_bpf`, is caused by rust code errors, but I don’t know the reason for other errors.

**Comment by yunwei37** [comment]
I have created a branch for include C API error, maybe you can continue on that branch?

https://github.com/eunomia-bpf/eunomia-bpf/tree/fix-wrapper

**Comment by yunwei37** [comment]
Fixed in #139
